symfony/plugins/orangehrmPimPlugin/Api/EmpEmergencyContactAPI.php

```diff
@@ -0,0 +1,303 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+
+namespace OrangeHRM\Pim\Api;
+
+
+use OrangeHRM\Core\Api\CommonParams;
+use OrangeHRM\Core\Api\V2\CrudEndpoint;
+use OrangeHRM\Core\Api\V2\Endpoint;
+use OrangeHRM\Core\Api\V2\Exception\RecordNotFoundException;
+use OrangeHRM\Core\Api\V2\Model\ArrayModel;
+use OrangeHRM\Core\Api\V2\ParameterBag;
+use OrangeHRM\Core\Api\V2\RequestParams;
+use OrangeHRM\Core\Api\V2\Serializer\EndpointCreateResult;
+use OrangeHRM\Core\Api\V2\Serializer\EndpointDeleteResult;
+use OrangeHRM\Core\Api\V2\Serializer\EndpointGetAllResult;
+use OrangeHRM\Core\Api\V2\Serializer\EndpointGetOneResult;
+use OrangeHRM\Core\Api\V2\Serializer\EndpointUpdateResult;
+use OrangeHRM\Core\Api\V2\Serializer\NormalizeException;
+use OrangeHRM\Core\Api\V2\Validator\ParamRule;
+use OrangeHRM\Core\Api\V2\Validator\ParamRuleCollection;
+use OrangeHRM\Core\Api\V2\Validator\Rule;
+use OrangeHRM\Core\Api\V2\Validator\Rules;
+use OrangeHRM\Core\Exception\DaoException;
+use OrangeHRM\Core\Exception\SearchParamException;
+use OrangeHRM\Core\Exception\ServiceException;
+use OrangeHRM\Entity\EmpEmergencyContact;
+use OrangeHRM\Entity\Employee;
+use OrangeHRM\Pim\Api\Model\EmpEmergencyContactModel;
+use OrangeHRM\Pim\Dto\EmpEmergencyContactSearchFilterParams;
+use OrangeHRM\Pim\Service\EmpEmergencyContactService;
+use OrangeHRM\Core\Traits\UserRoleManagerTrait ;
+
+class EmpEmergencyContactAPI extends Endpoint implements CrudEndpoint
+{
+    public const PARAMETER_EMP_NUMBER = 'empNumber';
+    public const PARAMETER_EEC_SEQNOS = 'seqNos';
```

Comment:
will use `CommonParams::PARAMETER_IDS` and `CommonParams::PARAMETER_ID` no need to expose API as seqNo.
You can refer `EmployeeDependentAPI`, I'll add comments what should be changed